src/usr/local/www/pkg.php

```diff
@@ -183,9 +183,9 @@ function domTT_title($title_msg) {
 
 // Breadcrumb
 if ($pkg['title'] != "") {
-	if (!$only_edit) {
- 		$pkg['title'] = $pkg['title'] . '/Edit';
-	}
+	/*if (!$only_edit) {						// Is any package still making use of this?? Is this something that is still wanted, considering the breadcrumb policy https://redmine.pfsense.org/issues/5527
```

Comment:
@sbeaver-netgate $only_edit is not set anywhere in pkg.php.
this means that, this code will always add '/Edit' to the title. Is that the way it is supposed to be then ?